README.md

```diff
@@ -245,6 +245,8 @@ class EofCallbackStream implements StreamInterface
 
     private $callback;
 
+    private $stream;
```

Comment:
Making this private won't work. We probably need to fix the stream decorator trait.

Reply:
Hmmm, maybe it will.

Reply:
I think you could add it to the trait, and then remove the `@property StreamInterface $stream` too, then you don't need to add it to very implementation.

But as per the commit you linked below, adding a private property is what was done to the rest of the codebase anyway.